pandas/tests/frame/conftest.py

```diff
@@ -110,6 +110,18 @@ def mixed_int_frame():
     return df
 
 
+@pytest.fixture
+def mixed_type_frame():
```

Comment:
so this fixtures itself is prob ok here. as we *know* that this is used in many places.

Reply:
on 2nd thought, let's see how often this is actually used, so move into the test module

Reply:
I'll reiterate from [another thread](https://github.com/pandas-dev/pandas/pull/25633#pullrequestreview-212612585), that with these fixturization PRs, I will:
* keep the `TestData` "fixtures" in `tests.frame.conftest`
* any new fixtures will stay in their respective modules
* the effort for #22471 is strictly translation, no breakup of tests
* [reorg of fixtures from `TestData` into submodules to happen later]